chart/tests/test_pgbouncer.py

```diff
@@ -427,7 +453,7 @@ def test_default_exporter_secret(self):
     def test_exporter_secret_with_overrides(self):
         connection = self._get_connection(
             {
-                "pgbouncer": {"enabled": True},
+                "pgbouncer": {"enabled": True, "metricsExporterSidecar": {"sslmode": "require"}},
```

Comment:
The expected value down on line 470 needs to be changed to `require`. I can't suggest it through the UI.

Reply:
Yesterday i applied all the suggestion.

Reply:
As I said, I can't suggest it through the UI - it's too far from other changes.

I've just pushed a commit that fixes it.